sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/timeExpressions.scala

```diff
@@ -233,3 +233,62 @@ object MinuteExpressionBuilder extends ExpressionBuilder {
     }
   }
 }
+
+// scalastyle:on line.size.limit
```

Comment:
It is not needed, it seems. Could you remove it, please.

Reply:
Yes done @MaxGekk